### PR TITLE
remove using namespace std;

### PR DESCRIPTION
**Pull request by x0rld**
using namespace std is a bad pratice.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1263"></a>

### CODE REVIEW

**Comment by xgupta** [comment]
Use std::vector instead of VLA to pass the clang-tidy errors. 
For example replace int n[] to std::vector<int> n;

**Comment by x0rld** [comment]
I will continue later

File: operations_on_datastructures/array_left_rotation.cpp [hunk 1]
```diff
@@ -22,9 +25,9 @@ int main() {
             }
         }
     }
-    cout << "Your rotated array is=\t";
+    std::cout << "Your rotated array is=\t";
     for (int j = 0; j < n; j++) {
-        cout << a[j] << " ";
+        std::cout << a[j] << " ";
     }
     getchar();
```
Comment (kvedala) [inline operations_on_datastructures/array_left_rotation.cpp:32]:
```suggestion
```

**Review by kvedala — CHANGES_REQUESTED** [review]
thank you for your contribution

**Comment by kvedala** [comment]
This folder and the code therein is not making much sense to me for this repo. @Panquesito7 @fhlasek @ayaankhan98  thoughts?

**Comment by fhlasek** [comment]
@kvedala I agree with you. On top of it, the code is of such a low quality that I don't think it carries much value. Rather than trying to refactor, I'd suggest to deprecate it and let the community recreate whatever valuable algorithms there might have been following the improved guidelines and best coding practices.

**Comment by kvedala** [comment]
@x0rld Thank you for your contribution. However, the admins find that this folder and the code therein is rudimentary and will need to be either removed altogether or salvaged in to one of the other folders in a more refined code. 

We more than welcome you to contribute in the latter 😄

**Pull request closed without merging**